internal/restore/volumesnapshotclass_restorer.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
This is pretty much a noop plugin atm.
Keeping this in this file as a place holder for now.